Toasters/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.2/css/all.min.css" />
+    <link rel="stylesheet" href="./documentation.css" />
+    <link rel="stylesheet" href="./index.css" />
+    <title>Notification Toasters</title>
+</head>
+
+<body>
+    <div class="container">
+        <div class="toaster success-toaster ">
+            <span class="toaster-icon"><i class="fas fa-check-circle"></i></span>
```

Comment:
you have added the icon, but it's not showing in the screenshot you have attached. Do check that.

Reply:
changed screenshot plz check.

Reply:
yes, it is showing now.